internal/driver/cli.go

```diff
@@ -45,8 +45,8 @@ type source struct {
 func parseFlags(o *plugin.Options) (*source, []string, error) {
 	flag := o.Flagset
 	// Comparisons.
-	flagBase := flag.StringList("base", "", "Source for base profile for profile subtraction")
-	flagDiffBase := flag.StringList("diff_base", "", "Source for diff base profile for comparison")
+	flagBase := flag.StringList("base", "", "Source of base profile for profile subtraction")
```

Comment:
Nit: you list base flag here first and diff_base second, but in the help message it's reversed. List diff_base flag here first to emphasize that's the flag most people should use?

Reply:
Done.